superset-frontend/stylesheets/antd/index.less

```diff
@@ -21,10 +21,6 @@
 @import '~antd/lib/style/mixins/index';
 @import '~antd/lib/style/core/base';
 
-*[class*='ant-'] {
```

Comment:
I removed this import, because *[class*='ant-'] selector gives global antd styles higher specifity than our styles and margins, font sizes etc. for some components were overriden. For example the margins of button components were reset to 0 when placed inside of popovers. I haven't found a valid use case of these styles so it seems to me that it's safe to delete them. However, I would appreciate a confirmation or a comment if we can't simply remove them.

Reply:
I *think* the reason this was included is so that the color variables below it in the code are inserted into the global styles. Can you confirm whether or not these colors are still being applied to elements correctly?

That said, I'm _really_ hoping to kill off LESS variables at some point, so everything uses variables pulled in from the Emotion Theme... maybe this is the time to start looking at a better way (Emotion or otherwise) to import/use the Theme colors for all the AntD components we're now using. Thoughts?